cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -100,6 +100,10 @@ func extractExecutionState(
 				RWF:   reportFileWriterFactory,
 			},
 			"newFungibleTokenTracker": reporters.NewFungibleTokenTracker(log, reportFileWriterFactory, chain, []string{reporters.FlowTokenTypeID(chain)}),
+			"atree": &reporters.AtreeReporter{
+				Log: log,
+				RWF: reportFileWriterFactory,
+			},
```

Comment:
How time-consuming is this reporter?  have we test it on mainnet data yet?

Reply:
@ramtinms I just confirmed it takes about 4.5 mins on the benchnet using May 13 mainnet checkpoint file.  This PR uses concurrency because it used to take a lot longer.

If this isn't fast enough, I'm wondering if we can run this report concurrently while other reports are running.

Reply:
Is this report done before the state extraction? 

I wonder if possible we can move this report post state extraction.

Reply:
> Is this report done before the state extraction?

@zhangchiqing I think this report is done [after the state extraction](https://github.com/onflow/flow-go/blob/master/ledger/complete/ledger.go#L415-L420) (after new root checkpoint file is created).